docs/DEVELOPING.md

```diff
@@ -180,13 +182,18 @@ The connector uses the Elastic ingest attachment processor plugin for extracting
 
 ### E2E Tests
 
-The framework allows users to test the connector end to end. To perform e2e test for Amazon S3 connector, run the following make command:
+The end to end test is usually performed by developers after the functional and system testing is completed. The framework allows users to test the connector end to end. To perform e2e test for Amazon S3 connector, run the following make command:
 ```shell
 $ make ftest NAME=s3
 ```
 
+ℹ️ Users do not need to have a running Elasticsearch instance or an Amazon S3 source to run this test. The docker compose file manages the complete setup of the development environment, i.e. both the mock Elastic instance and mock Amazon S3 source using the docker image.
+
 ℹ️ The e2e test uses default values defined in [configure Amazon S3 connector](#configure-amazon-s3-connector)
 
+## General Configuration
+The details of Elastic instance and other relevant fields such as `service` and `source` needs to be provided in the [config.yml](https://github.com/elastic/connectors-python/blob/8.6/config.yml) file. For more details check out the following [documentation](https://github.com/elastic/connectors-python/blob/8.6/docs/CONFIG.md).
```

Comment:
This change covers the general configuration which is nice (thx!).  For point 4 of https://github.com/elastic/connectors-python/issues/360, it will also be helpful to explain how S3 specific configurations like `buckets` can be set (https://github.com/elastic/connectors-python/issues/360#issuecomment-1404488683, i.e. an UI editor will appear on the UI once the first successful connection can be made).

Reply:
Updated with commit:
[resolve comments](https://github.com/elastic/connectors-python/pull/385/commits/f05b2b2b7fda74556d664f89d2092a072dc21a85)